Entity/ProcessType.php

```diff
@@ -46,11 +46,18 @@ class ProcessType extends AbstractTaxonomy
      */
     private $services;
 
+    /**
+     * @ORM\Column(type="float")
+     */
+    private $netDefaultValue;
+
     public function __construct()
     {
         $this->processes = new ArrayCollection();
         $this->processStatuses = new ArrayCollection();
         $this->services = new ArrayCollection();
+
+        $this->netDefaultValue = .7;
```

Comment:
Where does the value `.7` come from? I appreciate that this value will probably never change and it will be troublesome to get it from some configuration, but we should consider getting stuff like this from configuration.

Reply:
Good point.
It is configurable on the process type.
I'll move this default value to config.

Reply:
Fixed in: https://github.com/aakb/kontrolgruppen-core-bundle/pull/34/commits/70dcb1fe6ccc2cada3c44731ad56aed46ce80de0